src/app/recommendations.ts

```diff
@@ -62,7 +62,7 @@ export const RECOMMENDATIONS: Step[] = [
   { possibleIn: 600, necessaryAsOf: 700, level: ApplicationComplexity.Basic, step: 'update to RxJS 6', action: 'Remove deprecated RxJS 5 features using <a href="https://github.com/ReactiveX/rxjs-tslint" target="_blank">rxjs-tslint auto update rules</a><br/><br/>For most applications this will mean running the following two commands:<br/><br/>`npx rxjs-tslint`<br/>`rxjs-5-to-6-migrate -p src/tsconfig.app.json`'},
   { possibleIn: 600, necessaryAsOf: 800, level: ApplicationComplexity.Medium, step: 'remove rxjs-compat', action: 'Once you and all of your dependencies have updated to RxJS 6, remove `rxjs-compat`.'},
   { possibleIn: 610, necessaryAsOf: 800, level: ApplicationComplexity.Medium, step: 'use files instead of versionedFiles', action: 'If you use the Angular Service worker, migrate any `versionedFiles` to the `files` array. The behavior is the same.'},
-  { possibleIn: 700, necessaryAsOf: 700, level: ApplicationComplexity.Basic, step: 'TypeScript 3.1', action: 'Angular now uses TypeScript 3.1, read more about any potential breaking changes: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html', },
+  { possibleIn: 700, necessaryAsOf: 700, level: ApplicationComplexity.Basic, step: 'TypeScript 3.1', action: 'Angular now uses TypeScript 3.1, read more about any potential breaking changes: <a href="https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-1.html" target="_blank">TypeScript 3.1 release notes</a>', },
   { possibleIn: 700, necessaryAsOf: 700, level: ApplicationComplexity.Basic, step: 'Node 10', action: 'Angular has now added support for Node 10: https://nodejs.org/en/blog/release/v10.0.0/', },
```

Comment:
Please do the same for this link for Node as well.

Reply:
Thanks for the review, I've updated my PR.
Also, seems like external links are missing the associated icon (see below) that is needed for the links to be accessible. Shall I add them in this PR, create another PR or accessibility is not an issue for now?

![main-qimg-c1584f882355c17aeadf6b0fc9e5e95f-lq](https://github.com/angular/angular-update-guide/assets/55321938/b04053b7-1ffc-48dc-8ffe-b682c4346055)

Reply:
It would be better to take this separately. Please create a new issue for this.